zcash_client_sqlite/src/lib.rs

```diff
@@ -608,7 +594,7 @@ mod tests {
 
     /// Create a fake CompactBlock at the given height, containing a single output paying
     /// the given address. Returns the CompactBlock and the nullifier for the new note.
-    pub(crate) fn fake_compact_block(
+    pub fn fake_compact_block(
```

Comment:
Why not delete this and use the function in `zcash_extras` since they are the same?

Reply:
done..tthanks